packages/manager/modules/adp/src/deploy/deploy.html

```diff
@@ -0,0 +1,71 @@
+<section>
+    <!-- page header -->
+    <oui-page-header
+        data-heading="{{ ::'adp_deploy_title' | translate }}"
+        data-description="{{ ::'adp_deploy_description' | translate }}">
+    </oui-page-header>
+    <!-- page content -->
+    <div class="mx-auto" style="max-width: 1200px;">
```

Comment:
`-- style`

Reply:
Style has been removed and replaced with bootstrap css classes